app/views/government_rulesets/show.html.erb

```diff
@@ -2,43 +2,102 @@
+        <dt class="govuk-summary-list__key">
+          Issue Count:
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @counter %>
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Criticality:
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @criticality %>
+        </dd>
+      </div>
+    </dl>
+
+    <br/><br/>
+
+    <% if @issues.count > 0 %>
+    <table class="govuk-table linting-results-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Results</caption>
       <thead class="govuk-table__head">
       <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Code</th>
-        <th scope="col" class="govuk-table__header">Line</th>
-        <th scope="col" class="govuk-table__header">Character</th>
-        <th scope="col" class="govuk-table__header">Problem</th>
+        <th scope="col" class="govuk-table__header">Criticality</th>
+        <th scope="col" class="govuk-table__header">Issue</th>
+        <th scope="col" class="govuk-table__header">Line Numbers</th>
         <th scope="col" class="govuk-table__header">Remedy</th>
       </tr>
       </thead>
       <tbody class="govuk-table__body">
-      <% @spectral_results.each do |result| %>
+      <% @issues.each do |issue| %>
         <tr class="govuk-table__row">
-          <th scope="row" class="govuk-table__header"><%= result['code'] %></th>
-          <td class="govuk-table__cell"><%= result['range']['start']['line'] %></td>
-          <td class="govuk-table__cell"><%= result['range']['start']['character'] %></td>
-          <td class="govuk-table__cell"><%= result['message'] %></td>
+          <td class="govuk-table__cell criticality-<%= issue[:criticality] %>"><%= issue[:criticality] %></td>
+          <td class="govuk-table__cell"><%= issue[:message] %></td>
+          <td class="govuk-table__cell">0</td>
           <td class="govuk-table__cell">
-            <a href="https://cddo-api-spec-tech-docs.herokuapp.com/api-linting-remediation.html ">Details</a>
-          </td>
+          <a href="<%= ENV["WIKI_URL"] %>">Details</a></td>
         </tr>
       <% end %>
       </tbody>
     </table>
+    <% end %>
 
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-one-half">
+      <div class="govuk-grid-column-one-third">
         <%= govuk_link_to "Select Another Linter", rulesets_path %>
       </div>
-      <div class="govuk-grid-column-one-half">
+      <div class="govuk-grid-column-one-third">
         <%= govuk_link_to "Start Again", root_path %>
       </div>
+      <div class="govuk-grid-column-one-third">
+          <button class="govuk-button" data-module="govuk-button" onclick="window.print();">
+            Print or Save
+          </button>
+      </div>
     </div>
 
+    <details class="govuk-details" data-module="govuk-details">
```

Comment:
[suggestion]
I'd also no-print these details.

Reply:
This is also in.